### PR TITLE
clientside js for following likes and watchlist

### PR DESCRIPTION
**Pull request by parth-panchal**
What I've done for the movieId and followingId is that in the routes, I've added a check to see if it exists in the array we're removing it from, so even if a user decides to change the html and change the Id, it should not pass through ideally. I couldn't figure out how else to do it honestly.

### CODE REVIEW

File: views/profile.handlebars [hunk 1]
```diff
@@ -18,8 +18,8 @@
             <li><a href="/profile/statistics">Statistics</a></li>
         </ul>
     </nav>
-    <script></script>
     {{> (lookup . 'script_partial') }}
+    <script src="/public/js/script_profile.js"></script>
```
Comment (parshsee) [inline views/profile.handlebars]: You don't need to include the script here, it's already included in main

File: public/js/script_profile.js [hunk 2]
```diff
@@ -1,3 +1,71 @@
+//following
```
Comment (parshsee) [inline public/js/script_profile.js:1]: If you're not using jQuery, you could remove (function ($) {})(window.jQuery); from each of these

Comment (parth-panchal) [reply public/js/script_profile.js:1]: removed the jQuery thing, did I do it correctly?

Comment (parshsee) [reply public/js/script_profile.js:1]: you don't need the (function ($){})() part either,

Comment (parth-panchal) [reply public/js/script_profile.js:1]: no but i've kept it inside those boxes cause they all share the same variable name for form. I'd suggest we keep it as it is unless it is interfering with anything else on the codebase

Comment (parth-panchal) [reply public/js/script_profile.js:1]: no but i've kept it inside those wrappers cause they all share the same variable name for form. I'd suggest we keep it as it is unless it is interfering with anything else on the codebase

**Review by parshsee — APPROVED** [review]

**Pull request merged**